material.inc.php

```diff
@@ -1922,7 +1922,7 @@
 185 => array('name' => clienttranslate('Parnell Pitch Drop'),
     'i_demand_effect_1' => null,
     'i_demand_effect_1_is_compel' => false,
-    'non_demand_effect_1' => clienttranslate('Draw and meld a card of value one higher than the highest top card on your board. If the melded card has three ${icon_6}, you win.'),
+    'non_demand_effect_1' => clienttranslate('Draw and meld a card of value one higher than the highest top card on your board.  If the melded card has 3 ${icon_6}, you win.'),
```

Comment:
Can you please remove this extra space that you added?

Reply:
Can you change this "3" back to "three", please?

I'm curious what you touched this file haha.

Reply:
oh it looks like someone else fixed the text.  we can ignore the change to this file.

Reply:
Okay I reverted it.